DotNetRu.Android/MainActivity.cs

```diff
@@ -84,15 +84,18 @@ public MainActivity()
         {
             LocaleUtils.UpdateConfig(this);
         }
+
         public static MainActivity Current { get; }
 
         // GoogleApiClient client;
         protected override void OnCreate(Bundle savedInstanceState)
         {
+            SetTheme(Resource.Style.MyTheme);
```

Comment:
Why it's needed? This should be applied by default

Reply:
Exactly not. App is launched with `Theme = "@style/SplashTheme"` (see [line 34](https://github.com/DotNetRu/App/pull/55/files/82374bd5afe9ecf6c0763fced388baadc2d0de3b#diff-e8d186313ce687a878a76623be2f8b59R34)) and then set theme to main. See [this arcticle](https://android.jlelse.eu/launch-screen-in-android-the-right-way-aca7e8c31f52) for more information

Reply:
OK, I see, thanks!